ern-local-cli/src/commands/cauldron/add/nativeapp.js

```diff
@@ -17,10 +17,6 @@ exports.desc = 'Add a native application to the cauldron'
 
 exports.builder = function (yargs: any) {
   return yargs
-  .option('platformVersion', {
```

Comment:
This one should be kept as an option and not a positional argument.
Just remove `[platformVersion]` in the command syntax.

Reply:
https://github.com/yargs/yargs/blob/master/docs/advanced.md#positional-arguments
Yup make sense